app/assets/javascripts/components/alerts/campaign_alerts.jsx

```diff
@@ -19,6 +19,20 @@ class CampaignAlerts extends React.Component {
   }
 
   componentDidMount() {
+    // This clears Rails parts of the previous pages, when changing Campagn tabs
```

Comment:
What happens when you navigate to Alerts and then back to Overview?

Reply:
If that's a problem, consider hiding things with css (display: none) and then removing that css when they need to be shown again.

Reply:
That works fine. The only problem was when navigating from the React-Rails tab to React tab. Rails parts just stay under, so I used this to clear them if any. 
Navigating between two Rails-React tabs and React only to Rails-React was ok all the time.